### ISSUE TITLE
The date in To field can be set older than date in From field (Media Gallery, Uploaded Date or Modification Date filters) 

### ISSUE THREAD

**Issue by chalov-anton** [magento/adobe-stock-integration#880]
### Preconditions (*)
1. The issue will be valid after merging https://github.com/magento/adobe-stock-integration/pull/843
2. Install Magento with Adobe Stock Integration
3. Configure the integration in Stores -> Configuration -> System -> Adobe Stock Integration fieldset

### Steps to reproduce (*)
1. Go to **Media Gallery Development**, click **Filters**
2. Set, **Uploaded Date** or **Modification Date** _from_, for example Todays date
3. Try to set, **Uploaded Date** or **Modification Date** _to_, for example, a day before
4. Click **Apply Filters**
### Expected result (*)
1. There should not be an ability to select a date **To** older than the date in **From**
Like for example in _Marketing -  Cart Price Rules_. The older date is dimmed 
![filters_in_catalog price rule](https://user-images.githubusercontent.com/45624059/71469961-60923a00-27d3-11ea-8d60-990f2bc600e2.gif)

### Actual result (*)
1. The Filter is Applied  
![filters_in_media_gal](https://user-images.githubusercontent.com/45624059/71469607-68051380-27d2-11ea-96d0-5caf6716663e.gif)

**Comment by konarshankar07** [comment]
Hello guys,
I found that this bug is in magento repo and it can be reproduced in sale order grid also. Please check the below screenshot
![image](https://user-images.githubusercontent.com/39480008/71558742-770eee80-2a7c-11ea-9827-44e61b9bedcd.png)
Thanks

**Comment by sivaschenko** [comment]
@chalov-anton can you please check this bug on Magento grids and if it's reproducible - rephrase it to use Magento grid filters instead of media gallery  and move it to magento2 repo
